dataland-frontend/src/assets/css/main.css

```diff
@@ -0,0 +1,8 @@
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+  margin-top: 60px;
+}
```

Comment:
in `assets/images/logos` gibt es paar png Dataein, die nicht benuzut wurden

Reply:
Diese files sind für die responsiveness